witness/golang/omniwitness/internal/omniwitness/omniwitness.go

```diff
@@ -135,11 +152,28 @@ func Main(ctx context.Context, signer note.Signer, httpListener net.Listener, ht
 		c, f := c, f
 		// Continually feed this log in its own goroutine, hooked up to the global waitgroup.
 		g.Go(func() error {
+			glog.Infof("Feeder %q goroutine started", c.Origin)
+			defer glog.Infof("Feeder %q goroutine done", c.Origin)
 			return f(ctx, c, bw, httpClient, feedInterval)
 		})
 	}
 
-	// TODO(mhutchinson): Start the distributors too if auth details are present.
+	if len(operatorConfig.GithubUser) > 0 {
```

Comment:
nit: I think go style might prefer `if opCfg.GitHubUser != ""` since this `len` is really checking for empty/not-empty

Reply:
Done.